docs/sources/reference/stdlib/file.md

```diff
@@ -0,0 +1,24 @@
+---
+canonical: https://grafana.com/docs/alloy/latest/reference/stdlib/file/
+description: Learn about file functions
+menuTitle: file
+title: file
+---
+
+# file
+
+The `file` namespace contains functions related to files.
+
+## file.path_join
+
+The `file.path_join` function joins any number of path elements into a single path, separating them with an OS specific separator.
+
+### Examples
+
+```
+> file.path_join()
+""
+
+> file.path_join("this/is", "a/path")
```

Comment:
```suggestion
file.path_join()
""

file.path_join("this/is", "a/path")
```

Is the greater-than symbol representing the OK command line or part of the actual command.

Reply:
we use it in the other stdlib doc, I guess it's just a style to show input and output